src/manipulateToDo.js

```diff
@@ -45,20 +82,153 @@ const drop = (ev) => {
 
   if (prevNextIds.prevId < prevNextIds.nextId) {
     setTimeout(() => {
-      document.getElementById(id).style.backgroundColor = '#4c3c3c';
+      let elUnder = document.getElementById(id);
+      // document.getElementById(id).style.backgroundColor = "#4c3c3c";
+      const classArray = elUnder.classList.value.split(' ');
+      if (
+        classArray.includes('fa-check')
+        || classArray.includes('checkbox')
+        || classArray.includes('to-do')
+      ) {
+        elUnder = elUnder.parentElement.parentElement;
+      } else if (
+        classArray.includes('fa-arrows-alt')
+        || classArray.includes('two')
+      ) {
+        elUnder = elUnder.parentElement;
+      }
+
```

Comment:
This same code is repeating through this file several times. Making it a separate function would improve readability and stick a bit better to the YAGNI, what do you think? 🤓